src/styles/general.css

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  box-sizing: border-box;
```

Comment:
maybe there's no need to duplicate this rule

Reply:
exactly! fixed